superset-frontend/src/SqlLab/components/ResultSet.tsx

```diff
@@ -205,6 +227,17 @@ export default class ResultSet extends React.PureComponent<
     ) {
       this.fetchResults(nextProps.query);
     }
+    if (
```

Comment:
is there any way to close the alert and then have it update props without running a new query?